meetings/2024-05-29.md

```diff
@@ -0,0 +1,95 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2024-05-29T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Agenda
```

Comment:
```suggestion
* Status: Draft
```

"Agenda" is typical value for document status. "Agenda" is more like a document type. "Draft" or "Published" are common values for document status. This should remain "Draft" until "Published" (once updated with minutes and PR reviewed)

Reply:
Agree, I think "Agenda" is more of a category than a status.